### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by jmweir**

### CODE REVIEW

File: unanet-summarizer.js [hunk 1]
```diff
@@ -1,142 +1,156 @@
...
-var docTemplateGeneration = function(hoursByProjectTypeArray, totalPlus, totalNonPlus, totalHours){
-
-    var resultTable = document.createElement('table');
-    resultTable.setAttribute('style', 'border: 2px solid;');
+    return function() { 
+        var timeEntries = obtainTimeEntryRows();        
+        var reducers = getReducers();
 
-    var tableHeader = document.createElement('thead');
+        var properties = Object.keys(reducers).reduce(function(acc, property) {
```
Comment (SeanKilleen) [inline unanet-summarizer.js]: @jmweir do you think this impacts the readability? Maybe I'm just not in JS enough, but it reads a little cryptic to me.

**Comment by SeanKilleen** [comment]
I think this is 👍 to merge. Definitely more condensed, and the IIFE makes sense. A little concerned about readability in general but the trade-off seems fine here. I'll likely add some comments just to make it more accessible for newcomers.

**Comment by SeanKilleen** [comment]
(also, thanks!)

File: unanet-summarizer.js (hunk 1 above)
Comment (jmweir) [reply unanet-summarizer.js]: I think readability is a good callout. This can be made more readable, will submit with next PR.

**Pull request merged**